cmd/trust-manager/app/options/options.go

```diff
@@ -174,4 +189,10 @@ func (o *Options) addWebhookFlags(fs *pflag.FlagSet) {
 		"Directory where the Webhook certificate and private key are located. "+
 			"Certificate and private key must be named 'tls.crt' and 'tls.key' "+
 			"respectively.")
+	fs.IntVar(&o.Webhook.MinTlsVersion,
+		"min-tls-version", tls.VersionTLS12,
+		"Minimal TLS version to serve webhook.")
+	fs.StringVar(&o.Webhook.TlsCipherSuites,
+		"tls-cipher-suites", defaultCipherSuites,
```

Comment:
I don't think we should default the cipher suites explicity. Instead we should use the default in Go if no flag given - as we do right now.